app/components/post_textbox/post_textbox.js

```diff
@@ -272,7 +272,13 @@ export default class PostTextbox extends PureComponent {
     };
 
     handleUploadFiles = (images) => {
-        this.props.actions.handleUploadFiles(images, this.props.rootId);
+        /*eslint-disable */
+        const asciiFreeImages = images.map((image) => ({
+            ...image,
+            fileName: image.fileName.replace(/[^\x00-\x7F]/g, '')
+        }));
+        /*eslint-enable */
```

Comment:
I am open to suggestions on the eslint issue. 

re: https://eslint.org/docs/rules/no-control-regex

I think this is appropriate use of control characters here though because of what I am matching for. It is the simplest way to catch the 127 ascii characters

Reply:
What about adding an utils function to atrip those caracters?

Reply:
Is just removing the characters ideal, or could we escape them? [The example on this page](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent) looks like it might do that, but it's possible that the server may need help decoding them

Reply:
we also have this https://github.com/mattermost/mattermost-mobile/blob/master/app/utils/latinise.js I don't know if that can be of help

Reply:
ok. I will switch to one of those and test again. I will try the current util first. :)

Reply:
Sorry about the delayed response. For some reason my responses from last week were stuck *pending*